internal/handlers/handlers.go

```diff
@@ -1 +1,63 @@
 package handlers
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+// Парсим html-форму из файла index.html
+func IndexHandler(w http.ResponseWriter, r *http.Request) {
+	file, err := os.Open("index.html")
+	if err != nil {
+		http.Error(w, "Failed to open file", http.StatusInternalServerError)
+		return
+	}
+	defer file.Close()
+
+	w.Header().Set("Content-Type", "text/html")
+	w.WriteHeader(http.StatusOK)
+	io.Copy(w, file)
+}
+
+func UploadHandler(w http.ResponseWriter, r *http.Request) {
+	err := r.ParseMultipartForm(10 << 20)
+	if err != nil {
+		http.Error(w, "Error in form", http.StatusInternalServerError)
+		return
+	}
+
+	file, header, err := r.FormFile("myFile")
+	if err != nil {
+		http.Error(w, "Failed to get file", http.StatusInternalServerError)
+	}
+	defer file.Close()
+
+	data, err := io.ReadAll(file)
+	if err != nil {
+		http.Error(w, "Error reading file", http.StatusInternalServerError)
+		return
+	}
+
+	converted, err := service.Convert(string(data))
+	if err != nil {
+		http.Error(w, "Conversion error", http.StatusInternalServerError)
+	}
+
+	ext := filepath.Ext(header.Filename)
+	t := time.Now().UTC().Format("01022006_150405")
+	filename := fmt.Sprintf("%s%s", t, ext)
+
+	if err := os.WriteFile(filename, []byte(converted), 0644); err != nil {
+		http.Error(w, "Error writing file", http.StatusInternalServerError)
+		return
+	}
+	w.Header().Set("Content-Type", "text/plain")
```

Comment:
Чтобы не было кракозябр в браузере нужно так писать
```suggestion
	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
```